WordPressAuthenticator/Services/SignupService.swift

```diff
@@ -71,10 +71,13 @@ class SignupService {
                                     success(createdAccount, false, username, bearer_token)
         }, failure: { error in
             if let error = (error as NSError?) {
-                
+
+                // If an account already exists, the account email should be returned in the Error response.
+                // Extract it and return it.
                 var existingEmail = ""
-                if let errorData = error.userInfo[WordPressComRestApi.ErrorKeyErrorData] as? [String: String],
-                    let email = errorData.first?.value {
+                if let errorData = error.userInfo[WordPressComRestApi.ErrorKeyErrorData] as? [String: String] {
+                    let emailDict = errorData.first { $0.key == "email" }
```

Comment:
I like this! Could we create a static value for `email` instead of putting the literal here? Perhaps alongside `WordPressComRestApi.ErrorKeyErrorData`.

Reply:
@astralbodies @frosty - a new constant has been added to WPKit. Ready for another review please.